code/src/layouts/app-header/components/categories/Categories.scss

```diff
@@ -0,0 +1,17 @@
+.menu {
+  display: flex;
+  justify-content: flex-start;
+  padding: 10px 0;
+  background-color: #e9ecef;
+
+  &__item {
+    font-size: 16px;
+    color: #333;
+    cursor: pointer;
+    padding: 10px 20px;

```

Comment:
Same. Use `spacing` function. Maybe exact value of padding will be impossible to reflect with our fixed spacing system, but try to use values close to it